src/main/java/Calculator.java

```diff
@@ -0,0 +1,20 @@
+public class Calculator {
+
+    public double add(double a1, double a2){
+        return a1 + a2;
+    }
+
+    public double subtract(double a1, double a2){
+        return a1 - a2;
+    }
+
+    public double multiply(double a1, double a2){
+        return a1 * a2;
+    }
+
+    public double divide(double a1, double a2) throws IllegalArgumentException{
+        if (a2 == 0) throw new IllegalArgumentException("The divisor cannot be equal to 0");
```

Comment:
IF ELSE ELSE IF, FOR, WHILE etc always should be with braces.
So:
```
 if (b == 0) {
   throw new IllegalArgumentException("The divisor cannot be equal to 0");
 }
```
Two main reasons:
1. Readibility.
 Just compare:
 if (calcAvgMarginByX(45, arg2) || hasTrueBetween(89, 90)) calcAvgMarginByY(axis);
 
 and 
```
 if (calcAvgMarginByX(45, arg2) || hasTrueBetween(89, 90)) {
    calcAvgMarginByY(axis);
}   
```
 In first case we can't just clearly see when condition ends and when expression starts.
 
 2. Second reason it's harder to refactor, If I want to add additional logic , add one more line, I need first just to put braces and only then add my line:
 1st step:
  if (b == 0) {
   throw new IllegalArgumentException("The divisor cannot be equal to 0");
 }
 2nd step:
  if (b == 0) {
   addMyLine();
   throw new IllegalArgumentException("The divisor cannot be equal to 0");
 }
 
 A lot of bugs could be occured there, because it's not clearly where braces { } should be